src/platforms/common/usage/sdk-fingerprinting.mdx

```diff
@@ -33,16 +33,14 @@ You can use variable substitution to fill dynamic values into the fingerprint ge
 
 ## Group Errors With Greater Granularity
 
-Your application queries a Remote Procedure Call Model (RPC) interface or external Application Programming Interface (API) service, so the stack trace is generally the same (even if the outgoing request is very different).
+Let's imagine your application queries a Remote Procedure Call Model (RPC) interface or external Application Programming Interface (API) service, so the stack trace is generally the same (even if the outgoing request is very different).
```

Comment:
```suggestion
If your application queries a Remote Procedure Call Model (RPC) interface or external Application Programming Interface (API) service, the stack trace is generally the same, even if the outgoing request is very different.
```

Reply:
Hmm, the "problem" here is that this is really just an example use case of when this would be necessary - there may be others. So not 100% sure what the best wording is here... IMHO with `If` it sounds like "this is the case when you need to do that" 🤔

Reply:
that sounds great to me, thanks!